### PR TITLE
Changed links and names, added Mailing List

### PR DESCRIPTION
**Pull request by lindaicing**
Extension - changed link to the web store
“Be a beta tester” instead of “try Ethicli”
“Sign up for our mailing list”
Footer links, including terms of use

### CODE REVIEW

**Comment by lindaicing** [comment]
Added Mailing List
Screen Shot 2020-06-11 at 9 17 33 PM

When "Join" is clicked
Screen Shot 2020-06-11 at 9 17 38 PM

File: Website/index.html [hunk 1]
```diff
@@ -46,15 +46,11 @@ <h3>Ethicli helps you choose better products.</h3>
             <h2><span class="colorLGreen">About</span> <span class="colorCream">Ethicli</span></h2>
             <section class="half">
                 <article>
-                    <p>We are a team of developers who want to change the world by bringing you greater 
-                        consumer power! One of the best ways to vote for what you believe in is with your dollars. 
-                        Many of us can feel lost when it comes to knowing how our purchases
+                    <p>We are a team of developers who want to change the world by bringing you greater consumer power! One of the best ways to vote for what you believe in is with your dollars. Many of us can feel lost when it comes to knowing how our purchases
```
Comment (CubeOfCheese) [inline Website/index.html:49]: Change this back to multiple lines

File: Website/index.html [hunk 2]
```diff
@@ -46,15 +46,11 @@ <h3>Ethicli helps you choose better products.</h3>
...
                         affect others downstream, and we're here to help!</p>
                 </article>
                 <article>
-                    <p>Ethicli suggests better products while you shop by ranking how the product affects our workers, 
-                        planet, and community. Use our free browser extension to effortlessly shop from more ethical, 
-                        green, and smaller businesses!</p>
+                    <p>Ethicli suggests better products while you shop by ranking how the product affects our workers, planet, and community. Use our free browser extension to effortlessly shop from more ethical, green, and smaller businesses!</p>
```
Comment (CubeOfCheese) [inline Website/index.html:53]: and here

File: Website/index.html [hunk 3]
```diff
@@ -70,15 +66,13 @@ <h2><span class="colorLGreen">How to use</span> <span class="colorCream">Ethicli
                 </article>
                 <article class="imageDesc">
                     <h3>How it Works</h3>
-                    <p>Add some content for our curious users. Add some content for our curious users. 
-                        Add some content for our curious users. Add some content for our curious users.</p>
+                    <p>Add some content for our curious users. Add some content for our curious users. Add some content for our curious users. Add some content for our curious users.</p>
```
Comment (CubeOfCheese) [inline Website/index.html:69]: and here

File: Website/index.html [hunk 4]
```diff
@@ -70,15 +66,13 @@ <h2><span class="colorLGreen">How to use</span> <span class="colorCream">Ethicli
...
                 </article>
             </div>
             <div class="threes">
                 <article class="imageDesc">
                     <h3>About Ratings</h3>
-                    <p>Add some content for our curious users. Add some content for our curious users. 
-                        Add some content for our curious users. Add some content for our curious users.</p>
+                    <p>Add some content for our curious users. Add some content for our curious users. Add some content for our curious users. Add some content for our curious users.</p>
```
Comment (CubeOfCheese) [inline Website/index.html:75]: and here

**Pull request merged**